### ISSUE TITLE
More than one CSS class before colspan breaks table layout

### ISSUE THREAD

**Issue by a-kohout** [koreader/koreader#2849]
* KOReader version: v2015.11-1024-g3b813c7
* Device: Emulator

#### Issue
More than one css class before colspan breaks table layout.
Might be related to #2840 

Broken: `<td class="one two" colspan="2">`
![screen shot 2017-05-03 at 01 49 50](https://cloud.githubusercontent.com/assets/26367956/25644151/299941d0-2fa4-11e7-85fa-6ccee548c544.png)

Ok: `<td colspan="2" class="one two">`
![screen shot 2017-05-03 at 01 59 57](https://cloud.githubusercontent.com/assets/26367956/25644162/3cfc9466-2fa4-11e7-9b50-69aad7390462.png)

HTML:
```
  <table>
    <tbody>
      <tr>
        <td colspan="4">
          Lorem ipsum dolor sit amet, consectetur adipiscing elit.
        </td>
      </tr>

      <tr>
        <td class="one two" colspan="2">
          Column 1
        </td>

        <td class="one two" colspan="2">
          Column 2
        </td>
      </tr>

      <tr>
        <td>
          1.
        </td>

        <td>
          Text 1
        </td>

        <td>
          1.
        </td>

        <td>
          Text 1
        </td>
      </tr>

      <tr>
        <td></td>

        <td></td>

        <td>
          2.
        </td>

        <td>
          Text 2
        </td>
      </tr>

      <tr>
        <td></td>

        <td></td>

        <td>
          3.
        </td>

        <td>
          Text 2
        </td>
      </tr>
    </tbody>
  </table>
```

#### Steps to reproduce
[untitled.epub.zip](https://github.com/koreader/koreader/files/972219/untitled.epub.zip)

**Comment by frankyifei** [comment]
I think it is not supported by crengine now. so it is not a bug

**Comment by a-kohout** [comment]
If crengine ignored the second class it would be merely a bug in the CSS implementation since this violates spec.
However the CSS parser affecting the markup that follows is pretty severe as it's not just changing visuals but the representation of data.

**Comment by a-kohout** [comment]
Fixed in `v2015.11-1074-g9b557ed`
